tests/integration/bond_test.py

```diff
@@ -403,3 +406,77 @@ def test_change_bond_option_miimon(bond99_with_2_slaves):
     bond_options["miimon"] = "200"
     libnmstate.apply(desired_state)
     assertlib.assert_state(desired_state)
+
+
+def test_create_bond_without_mode():
+    with bond_interface(name=BOND99, slaves=[], create=False) as state:
+        state[Interface.KEY][0][Bond.CONFIG_SUBTREE].pod(Bond.MODE)
```

Comment:
We need to celebrate, the CI found an actual bug: ;-) :fireworks: 
https://travis-ci.com/nmstate/nmstate/jobs/289595195#L118
This should probably be `pop`